srcpkgs/python-pyglet/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'python-pyglet'
+pkgname=python-pyglet
+version=1.3.2
+revision=1
+wrksrc=pyglet-${version}
+build_style=python-module
+hostmakedepends="python-setuptools python3-setuptools"
+makedepends="python-devel python3-devel"
```

Comment:
replace with depends="python"

there are no compiled C extensions so no need for python headers.